fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -93,7 +102,15 @@ def self.available_options
                                        env_name: "FL_SPM_VERBOSE",
                                        description: "Increase verbosity of informational output",
                                        type: Boolean,
-                                       default_value: false)
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :simulator,
+                                       env_name: "FL_SPM_SIMULATOR",
+                                       description: "Specifies the simulator to pass for Swift Compiler",
```

Comment:
We usually add the available options in the description so they are printed in the documentation and users know them before they need to hit a failure in `verify_block` step 😊 See e.g.:

https://github.com/fastlane/fastlane/blob/7da098f32aa12edd9ec0c97b0b106b7006708282/fastlane/lib/fastlane/actions/spm.rb#L41-L47

```suggestion
                                       description: "Specifies the simulator to pass for Swift Compiler (one of: #{valid_simulators.join(', ')})",
```